css_tasks/task9/js/app.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
+
+// IMPORTS
+import {submitLoginForm, submitContactForm} from './forms.js';
+
+// HANDLERS
+const loginForm = document.forms.loginForm;
+const contactForm = document.forms.contactForm;
```

Comment:
any access to your DOM should be wrapped to DOMContentLoaded callback.
Not each one, but the whole script that's going to access DOM